### PR TITLE
Fixes #15810: Incorrect permission on /var/rudder/reports

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/15810

### CODE REVIEW

File: relay/sources/rudder-server-relay-postinst [hunk 1]
```diff
@@ -54,10 +54,10 @@ chown rudder: /var/rudder/shared-files
 chmod 770 /var/rudder/shared-files
 chmod 755 /var/rudder/inventories
 
-for dir in /var/rudder/inventories/incoming /var/rudder/inventories/accepted-nodes-updates /var/rudder/reports/incoming
+for dir in /var/rudder/inventories/incoming /var/rudder/inventories/accepted-nodes-updates /var/rudder/reports/incoming /var/rudder/reports/failed
 do
   chmod 770 ${dir}
-  chown rudder:${APACHE_GROUP} ${dir}
+  chown ${APACHE_USER}:rudder ${dir}
```
Comment (peckpeck) [inline relay/sources/rudder-server-relay-postinst]: why this change ?

Comment (amousset) [reply relay/sources/rudder-server-relay-postinst]: We need these folder to be writeable by rudder-relayd, which runs as `rudder-relayd:rudder`. Using the apache user and the Rudder group should make this work for both.
